ams/ams-server/src/main/java/com/netease/arctic/ams/server/utils/HiveLocationUtils.java

```diff
@@ -45,7 +46,7 @@ public static Set<String> getHiveLocation(ArcticTable table) {
         try {
           Table hiveTable = ((SupportHive) table).getHMSClient().run(client ->
               client.getTable(table.id().getDatabase(), table.id().getTableName()));
-          hiveLocations.add(hiveTable.getSd().getLocation());
+          hiveLocations.add(TableFileUtils.getUriPath(hiveTable.getSd().getLocation()));
```

Comment:
Why change to use `TableFileUtils.getUriPath()`, does it bring problems before the change?

Reply:
Because all file locations use TableFileUtils.getUriPath() to remove the URI, I think the Hive location also should use to avoid potential risks.